src/components/components/ReinforcementStatement/ReinforcementStatement.scss

```diff
@@ -0,0 +1,55 @@
+.Reinforcement {
+  display: flex;
+  flex-direction: row;
```

Comment:
flex-direction: row is the default value, so you dont need that here

Reply:
removed  flex-direction